legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/contract/storeContract.pure

```diff
@@ -190,19 +191,25 @@ function meta::relational::contract::planExecution(sq:meta::pure::mapping::Store
         $sq->meta::pure::graphFetch::executionPlan::planGraphFetchExecution($ext, $m->toOne(), $runtime->toOne(), $exeCtx, $extensions, $debug);,
 
       |
-        // Normal Flow
-       let connections = $runtime.connections->filter(c | $c->instanceOf(DatabaseConnection));
-       let storeRuntime = if($connections->size() <= 1,
-                             | $runtime->toOne(),
-                             | let oldRuntime = $runtime->toOne();
-                               let dbConn = $oldRuntime->connectionByElement($store)->cast(@DatabaseConnection);
-                               ^$oldRuntime(connections = $dbConn);
-                          );
-
-      let queryExeCtx = if($exeCtx->instanceOf(RelationalExecutionContext),|$exeCtx,|[])->cast(@RelationalExecutionContext);
-      let originalQuery = $sq.fe->toSQLQuery($m->toOne(), $sq.inScopeVars, $queryExeCtx, $debug, $extensions);
-      $originalQuery->postProcessSQLQuery($store, $ext, $m->toOne(), $storeRuntime, $exeCtx, $extensions)
-                    ->generateExecutionNodeForPostProcessedResult($sq, $store, $ext, $m->toOne(), $storeRuntime, $exeCtx, $debug, $extensions);
+        if($fe.func->in([meta::pure::mutation::execution::mutationFunctions()]),
```

Comment:
Same as a comment leave above, lets remove the wrapping `[]`.

Reply:
Done.